azurelinuxagent/common/protocol/wire.py

```diff
@@ -857,100 +857,77 @@ def check_wire_protocol_version(self):
-            msg = "[PERIODIC] Host plugin reconfigured with new parameters. " \
+            new_container_id, new_role_config_name = host_plugin.container_id, host_plugin.role_config_name
+            msg = "Host plugin reconfigured with new parameters. " \
                   "ContainerId: {0}, role config file: {1}.".format(new_container_id, new_role_config_name)
-            logger.periodic_info(logger.EVERY_SIX_HOURS, msg)
+            logger.info(msg)
 
             try:
                 ret = host_func()
-                if ret:
-                    msg = "[PERIODIC] Request succeeded using the host plugin channel after goal state refresh. " \
+                if ret is not None:
+                    msg = "Request succeeded using the host plugin channel after goal state refresh. " \
                           "ContainerId changed from {0} to {1}, " \
-                          "role config file changed from {2} to {3}.".format(old_container_id, new_container_id,
-                                                                             old_role_config_name, new_role_config_name)
-                    add_periodic(delta=logger.EVERY_SIX_HOURS,
-                                 name=AGENT_NAME,
-                                 version=CURRENT_VERSION,
-                                 op=WALAEventOperation.HostPlugin,
-                                 is_success=True,
-                                 message=msg,
-                                 log_event=True)
+                          "role config file changed from {2} to {3}.".format(old_container_id, new_container_id, old_role_config_name, new_role_config_name)
+                    add_event(AGENT_NAME, op=WALAEventOperation.HostPlugin, version=CURRENT_VERSION, is_success=True, message=msg, log_event=True)
 
-            except (ResourceGoneError, InvalidContainerError) as e:  # pylint: disable=C0103
-                msg = "[PERIODIC] Request failed using the host plugin channel after goal state refresh. " \
+            except Exception as error:
+                msg = "Request failed using the host plugin channel after goal state refresh. " \
                       "ContainerId changed from {0} to {1}, role config file changed from {2} to {3}. " \
-                      "Exception type: {4}.".format(old_container_id, new_container_id, old_role_config_name,
-                                                    new_role_config_name, type(e).__name__)
-                add_periodic(delta=logger.EVERY_SIX_HOURS,
-                             name=AGENT_NAME,
-                             version=CURRENT_VERSION,
-                             op=WALAEventOperation.HostPlugin,
-                             is_success=False,
-                             message=msg,
-                             log_event=True)
-                raise
+                      "Exception type: {4}.".format(old_container_id, new_container_id, old_role_config_name, new_role_config_name, type(error).__name__)
```

Comment:
why do we log the exception type and not the message?

Reply:
This is old behaviour, not sure. Let me check

Reply:
Leaving as-is for now. We never logged the full exception, possibly due to privacy? Not sure, keeping this the same to reduce the scope of changes in this PR.